ui/src/timeMachine/constants/visGraphics.tsx

```diff
@@ -614,4 +697,9 @@ export const VIS_GRAPHICS = [
     name: 'Table',
     graphic: GRAPHIC_SVGS[ViewType.Table],
   },
+  {
+    type: ViewType.Scatter,
+    name: 'Scatter',
+    graphic: GRAPHIC_SVGS[ViewType.Scatter],
```

Comment:
I think the scatterplot needs to be featured flagged in Cloud to start, so that we can ensure the API updates are rolled out before the UI updates. You can do that by adding a

```
    featureFlag: 'scatterplot',
```
line here and then adding the `scatterplot` feature flag to the constants here:

https://github.com/influxdata/influxdb/blob/ff07d6a60ee54aea5accd1a0e1a0bfa516b1cb13/ui/src/shared/utils/featureFlag.ts#L4-L10

Reply:
done!